src/Microsoft.AspNet.Hosting/Server/ServerLoader.cs

```diff
@@ -18,59 +18,29 @@ public ServerLoader(IServiceProvider services)
             _services = services;
         }
 
-        public IServerFactory LoadServerFactory(string serverFactoryIdentifier)
+        public IServerFactory LoadServerFactory([NotNull] string assemblyName)
```

Comment:
Methods implementing interfaces should use the same parameter names as the interface. `serverName` in this case.